src/EditorFeatures/Core/Implementation/Diagnostics/AbstractAggregatedDiagnosticsTagSource.ReadOnlyMode.cs

```diff
@@ -1,85 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@CyrusNajmabadi I am not sure why you removed these two mode. each one does simple stuff for their specific need and contraint without polluting code with concern each mode can't have.